packages/web-api/src/response/TeamInfoResponse.ts

```diff
@@ -24,6 +24,7 @@ export interface Team {
   email_domain?: string;
   icon?:         Icon;
   is_verified?:  boolean;
+  url?:          string;
```

Comment:
I'm just comparing what we have in these responses to what's on the docs site, and for [the `teams.info` method](https://api.slack.com/methods/team.info), `is_verified` and `url` does not exist, but it also says that `enterprise_id` and `enterprise_name` exist. Should we add the `enterprise_*` properties here, too, then, and should the `is_verified` and `url` properties be documented on the public site?

Reply:
>Should we add the enterprise_* properties here, too, 

Yes, we should. The reason why enterprise_* properties are not available in this class is that the Java SDK integration tests with team.info endpoint do with only a non-Grid workspace. I'll update the tests and this PR later on.

>then, and should the is_verified and url properties be documented on the public site?

Yes, the example should be updated.